drive/files.py

```diff
@@ -223,9 +230,10 @@ def download(self, writer, mime_type=None):
         """
         if not self._client:
             return False
-        return self._client.download(self.id, writer, mime_type=mime_type)
+        self._client.download(self.id, writer, mime_type=mime_type)
+        return None
```

Comment:
MyPy did not like returning the result of a function that always returns None, so I had to separate the `return` on a separate line (it needs an explicit return because there are multiple return types).